exchange-client.gemspec

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'openstax/exchange/client/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "openstax-exchange-client"
```

Comment:
@jpslav - changed to openstax_exchange

FWIW, hyphens vs underscores do carry meaning in gems: http://guides.rubygems.org/name-your-gem/